Github-copilot-skillmd-guide.html

```
@@ -0,0 +1,2094 @@
+        .sidebar-toggle {
+          display: block;
+        }
+        .ref-grid {
+          grid-template-columns: 1fr;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <div class="top-bar"></div>
+    <button class="sidebar-toggle" id="sidebarToggle" aria-label="目次を開閉">☰ 目次</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)